Directory.Build.targets

```diff
@@ -9,6 +9,24 @@
     <ImportDirectoryBuildTargets>false</ImportDirectoryBuildTargets>
   </PropertyGroup>
 
+  <!-- temporarily remap AssemblyKey to StrongNameKeyId -->
+  <PropertyGroup>
+    <StrongNameKeyId Condition="'$(AssemblyKey)' == 'MSFT'">Microsoft</StrongNameKeyId>
+    <StrongNameKeyId Condition="'$(AssemblyKey)' == 'ECMA'">ECMA</StrongNameKeyId>
+    <StrongNameKeyId Condition="'$(AssemblyKey)' == 'Open' or '$(IsTestProject)' == 'true'">Open</StrongNameKeyId>
+    <StrongNameKeyId Condition="'$(AssemblyKey)' == 'SilverlightPlatform'">SilverlightPlatform</StrongNameKeyId>
```

Comment:
Is that for Silverlight assemblies? Are we still building some in corefx?

Reply:
I'm not sure if we are building any in corefx it might only be coreclr but I added the mapping for completeness. I should file an issue to remove this mapping and simply replace all the places in file because I didn't want to do that just yet.

Reply:
Filed https://github.com/dotnet/corefx/issues/33112 for cleaning this up.